applications/ColossalQA/examples/webui_demo/server.py

```diff
@@ -77,12 +77,16 @@ def generate(data: GenerationTaskReq, request: Request):
         colossal_api = ColossalAPI(model_name, all_config["model"]["model_path"])
         llm = ColossalLLM(n=1, api=colossal_api)
     elif all_config["model"]["mode"] == "api":
-        all_config["chain"]["mem_llm_kwargs"] = None
-        all_config["chain"]["disambig_llm_kwargs"] = None
-        all_config["chain"]["gen_llm_kwargs"] = None
         if model_name == "pangu_api":
             from colossalqa.local.pangu_llm import Pangu
-            llm = Pangu(id=1)
+            
+            gen_config = {
+                "user": "User",
+                "max_tokens": all_config["chain"]["disambig_llm_kwargs"]["max_new_tokens"],
+                "temperature": all_config["chain"]["disambig_llm_kwargs"]["temperature"],
+                "n": 1
+            }
```

Comment:
Same that the `user` can also be customed?

What does the `n` mean here. Please also add comments here.

Thanks